conversions/hex-bin.py

```diff
@@ -0,0 +1,40 @@
+#code to convert hexadecimal into binary
+from doctest import testmod
+import math
+
+
+hex_str = input("Enter your value: ").strip()
+num : int = int(hex_str, 16)
+bin_str: str = ""
+
+
+"""
+Here, we have used " >> " bitwise operator
+Bitwise right shift: 
+Shifts the bits of the number to the right and fills 0 on voids left as a result. 
+Similar effect as of dividing the number with some power of two.
+Example: 
+a = 10
+a >> 1 = 5 
+
+"""
+
+def convert(num: int, bin_str: str) -> str:
+    """
+    >>> Enter your value: AC
+    >>> Your value in BINARY is 10101100
+    
+    """
+    while n>0:
```

Comment:
This would raise a `NameError` as the variable is not defined prior to it's use.